cores/esp32/HWCDC.cpp

```diff
@@ -324,14 +322,19 @@ void HWCDC::begin(unsigned long baud) {
 
   // Peripheral Manager setting for USB D+ D- pins
   uint8_t pin = USB_INT_PHY0_DM_GPIO_NUM;
+  if (perimanGetBusDeinit(ESP32_BUS_TYPE_USB_DM) == NULL) {
```

Comment:
is this check necessary? could the deinit callback be set to something else? If not, there is nothing wrong in just assigning it to the same thing

Reply:
then I am OK with it